fluentd/configs.d/openshift/filter-viaq-data-model.conf

```diff
@@ -9,4 +9,32 @@
   rename_time_if_missing "#{ENV['CDM_RENAME_TIME_IF_MISSING'] || false}"
   src_time_name "#{ENV['CDM_SRC_TIME_NAME'] || 'time'}"
   dest_time_name "#{ENV['CDM_DEST_TIME_NAME'] || '@timestamp'}"
+  <formatter>
+    tag "system.var.log**"
+    type sys_var_log
+    remove_keys host,pid,ident
+  </formatter>
+  <formatter>
+    tag "journal.system**"
+    type sys_journal
+    remove_keys log,stream,MESSAGE,_SOURCE_REALTIME_TIMESTAMP,__REALTIME_TIMESTAMP,CONTAINER_ID,CONTAINER_ID_FULL,CONTAINER_NAME,PRIORITY,_BOOT_ID,_CAP_EFFECTIVE,_CMDLINE,_COMM,_EXE,_GID,_HOSTNAME,_MACHINE_ID,_PID,_SELINUX_CONTEXT,_SYSTEMD_CGROUP,_SYSTEMD_SLICE,_SYSTEMD_UNIT,_TRANSPORT,_UID,_AUDIT_LOGINUID,_AUDIT_SESSION,_SYSTEMD_OWNER_UID,_SYSTEMD_SESSION,_SYSTEMD_USER_UNIT,CODE_FILE,CODE_FUNCTION,CODE_LINE,ERRNO,MESSAGE_ID,RESULT,UNIT,_KERNEL_DEVICE,_KERNEL_SUBSYSTEM,_UDEV_SYSNAME,_UDEV_DEVNODE,_UDEV_DEVLINK,SYSLOG_FACILITY,SYSLOG_IDENTIFIER,SYSLOG_PID
+  </formatter>
+  <formatter>
+    tag "kubernetes.journal.container**"
+    type k8s_journal
+    remove_keys "#{ENV['K8S_FILTER_REMOVE_KEYS'] || 'log,stream,MESSAGE,_SOURCE_REALTIME_TIMESTAMP,__REALTIME_TIMESTAMP,CONTAINER_ID,CONTAINER_ID_FULL,CONTAINER_NAME,PRIORITY,_BOOT_ID,_CAP_EFFECTIVE,_CMDLINE,_COMM,_EXE,_GID,_HOSTNAME,_MACHINE_ID,_PID,_SELINUX_CONTEXT,_SYSTEMD_CGROUP,_SYSTEMD_SLICE,_SYSTEMD_UNIT,_TRANSPORT,_UID,_AUDIT_LOGINUID,_AUDIT_SESSION,_SYSTEMD_OWNER_UID,_SYSTEMD_SESSION,_SYSTEMD_USER_UNIT,CODE_FILE,CODE_FUNCTION,CODE_LINE,ERRNO,MESSAGE_ID,RESULT,UNIT,_KERNEL_DEVICE,_KERNEL_SUBSYSTEM,_UDEV_SYSNAME,_UDEV_DEVNODE,_UDEV_DEVLINK,SYSLOG_FACILITY,SYSLOG_IDENTIFIER,SYSLOG_PID'}"
+  </formatter>
+  <formatter>
+    tag "kubernetes.var.log.containers**"
+    type k8s_json_file
+    remove_keys log,stream
+  </formatter>
+  <elasticsearch_index_name>
+    tag "journal.system** system.var.log** **_default_** **_openshift_** **_openshift-infra_** mux.ops"
+    name_type operations_full
+  </elasticsearch_index_name>
+  <elasticsearch_index_name>
+    tag "**"
```

Comment:
Will this also match the same tags as line 33?  In other words, how to records with tags match on line 33 avoid be processed again with this filter due to also matching on line 37?

Reply:
no - the way it works is that once it matches, there is no further processing - so once the match is hit on line 33, it will not go to line 37